modules/internal/localeModels/flat/LocaleModel.chpl

```diff
@@ -63,14 +63,14 @@ module LocaleModel {
     // that it is intended to represent.  This trick is used
     // to establish the equivalence the "locale" field of the locale object
     // and the node ID portion of any wide pointer referring to it.
-    proc LocaleModel() {
```

Comment:
Am I reading this correctly that this is another instance of a class type sharing the same name as the module in which it is defined?

Reply:
Correct.  Filename: LocaleModel.chpl, Module name: LocaleModel, class name: LocaleModel.

Reply:
We've run into problems with this in the past, and have updated other types/modules to have a different name.  Do we think that's something we should change here or do we want to wait until that actually becomes a problem?

Reply:
I have no objection to being proactive and taking care of it now.

Reply:
I think taking care of it now is smart, but that we should do it in a distinct PR since it isn't directly related to this one.